project/Dependencies.scala

```diff
@@ -6,23 +6,29 @@ import sbt.Keys._
 object Dependencies {
 
   object Versions {
-    val akka = "2.5.17"
-
-    val scalaTest = "3.0.5"
+    val akka = "2.6.4"
     val scalaJava8Compat = "0.9.0"
+    val alpakkaKafka = "2.0.2"
+    val scalaTest = "3.0.5"
   }
 
   object Compile {
     val akkaStream = "com.typesafe.akka" %% "akka-stream" % Versions.akka
+    val akkaSlf4j = "com.typesafe.akka" %% "akka-slf4j" % Versions.akka
+    val logback = "ch.qos.logback" % "logback-classic" % "1.2.3"
```

Comment:
slf4j and logback shouldn't be compile dependencies. We might want to add logback as test dependency (if we write tests with Akka Typed)